cargo-cyclonedx/src/cli.rs

```diff
@@ -37,6 +37,20 @@ pub struct Args {
     #[clap(long = "quiet", short = 'q')]
     pub quiet: bool,
 
+    // The feature selection flags are not mutually exclusive in Cargo,
+    // so we keep the same behavior here too.
+    /// Activate all available features
+    #[clap(long = "all-features")]
+    pub all_features: bool,
```

Comment:
I've copied the help messages from Cargo itself here.

I'll update the README before release, because otherwise I'll run into lots of merge conflicts. That text is not neatly split into lines, unlike code.